source/gamesavemenu.cpp

```diff
@@ -48,6 +49,8 @@ static bool reloadSave(MenuButton * button, void * param)
     // Close menu
     MenuWindow::sCurMenu->fullClose();
 
+    spm::mario::marioGetPtr()->motionId = 0;
```

Comment:
I think it might cause issues to just write here directly instead of calling marioChgMot

Reply:
Calling marioChgMot works; will add in next commit